### ISSUE TITLE
"a manifest file (anything that doesn't start with '<' will be considered a manifest file)"

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/echidna#89]
This instructions for manifest file don't make any sense to me:( Do you need me to create a file? I don't understand what this means: 
- "a manifest file (anything that doesn't start with '<' will be considered a manifest file)"

Please avoid passive voice (considered by who?).

**Comment by astorije** [comment]
@tripu, I agree that giving explanation on how the URL is detected to be a manifest file is unnecessary in the wikis. We can keep that in a dev-oriented wiki at most.

@marcoscaceres, the [manifest explanation that comes next](https://github.com/w3c/echidna/wiki/Preparing-your-document#manifest-file) seems clear to me, so maybe removing this statement is enough. After reading it, are you still being confused?

**Comment by marcoscaceres** [comment]
I'm still uncomfortable that the manifest is any file that doesn't start with "<" is a manifest. This might be the case, but maybe just recommend a file name? It makes it easier to follow along... my brain instantly goes "but what about if it's a binary file? what about if it hits a CSS file? do I need to put the manifest outside the folder that contains the spec?" etc.

**Comment by marcoscaceres** [comment]
I think the bigger problem with the "Preparing your document" section is that it doesn't follow the approach of "[How to get up and running](https://github.com/w3c/echidna#how-to-get-it-up-and-running)". I.e., a simple list of steps that an author must for to go from "zero to hero". 

In most cases, it seems like the only thing an editor needs to add is the Editor-IDs. However, no editor is going to do that manually, as no one (that I know, at least) writes specs by hand: they either use ReSpec or Bikeshed. So, maybe it would be best to tailor this section to ReSpec and BikeShed users. 

So,

## Preparing your document

### Respec
In Respec, you first add the `w3cid` to the Editor's collection in your ReSpec config. You can find your ID by going to https://www.w3.org/users/myprofile. After you log in, you will be redirected and your profile number will appear in the URL( e.g., https://www.w3.org/users/39125) . 

If there are multiple editor's, you will need each editor to send you their number using the method above.  

(NOTE: be nice if the editor number just showed up in the user's profile page. A way to search for editor's numbers would also be extremely helpful.)

```JS
 editors: [{
        name: "Jolly SpecEditor",
        w3cid: 19125
    }]
``` 

### Bikeshed
In Bikeshed, simply add .... 

### Manually 
You must be insane, but here we go anyway ... text about  `dd`s

And so on...

**Comment by astorije** [comment]
@marcoscaceres, regarding manifest detection and parsing, keep in mind that we went for the simplest version possible at the moment. We were aware that the current solution is temporary and flawed, but you raise some good points. The goal here was simply to find a quick way to detect manifest vs single HTML file without asking the requester to check a checkbox.
We'll take that into consideration when we build a better system for detecting and parsing.

Regarding the documentation, it is a task for @plehegar, who has been editing the wiki so far. Also keep in mind that this project is separate from ReSpec and Bikeshed (and actually, I have never used these tools myself!) so I'm not sure the Echidna doc should describe how to use them. But again, it will be @plehegar's call.

**Comment by plehegar** [comment]
I'm adding this one on my TODO list.

**Comment by tripu** [comment]
I'm hoping that darobin/echidna-manifester#2 will make it possible to get rid of manual manifests; Echidna would infer linked resources, and editors wouldn't bother enumerating files by hand.

**Comment by tripu** [comment]
Note to self: [`file-type`](https://www.npmjs.com/package/file-type) (h/t @deniak)

**Comment by tripu** [comment]
Documentation has improved since we had this discussion.

Anyway, I just edited the beginning of [that wiki page](https://github.com/w3c/echidna/wiki/Preparing-your-document) to clarify that bit about the char `<` and the &ldquo;HTML5 doc vs. manifest file&rdquo; distinction.

We are already using `file-type` in Echidna (although not for this).

[`w3c/echidna-manifester`](https://github.com/w3c/echidna-manifester) is abandoned in practice: it never got traction among editors; writing a manifest more or less &ldquo;by hand&rdquo; is a good way for editors to review what they're about to publish; having Echidna do that itself could conceal bugs, eg including unintended resources with the spec.

Closing this issue.
